maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -305,7 +308,108 @@ else if ((wabProperty != null) || hasWablibs) {
 					builder.setProperty(Constants.SNAPSHOT, TSTAMP);
 				}
 			}
+			
+			// Set Bundle-Description
+			if (builder.getProperty(Constants.BUNDLE_DESCRIPTION) == null) {
+				// may be null
+				if (StringUtils.isNotBlank(project.getDescription())) {
+					builder.setProperty(Constants.BUNDLE_DESCRIPTION, project.getDescription());
+				}
+			}
 
+			// Set Bundle-Vendor
+			if (builder.getProperty(Constants.BUNDLE_VENDOR) == null) {
+				if (project.getOrganization() != null && StringUtils.isNotBlank(project.getOrganization().getName())) {
+					builder.setProperty(Constants.BUNDLE_VENDOR, project.getOrganization().getName());
+				}
+			}
+			
+			// Set Bundle-License
+			if (builder.getProperty(Constants.BUNDLE_LICENSE) == null) {
+				List<String> licenses = new ArrayList<>();
+				for (License license : project.getLicenses()) {
+					StringBuilder licenseValue = new StringBuilder();
+					licenseValue.append("\"").append(license.getName()).append("\"");
+					// link is optional
+					if (StringUtils.isNotBlank(license.getUrl())) {
+						licenseValue.append(";link=\"").append(license.getUrl()).append("\"");
+					}
+					licenses.add(licenseValue.toString());
+				}
+				if (licenses.size() > 0) {
+					builder.setProperty(Constants.BUNDLE_LICENSE, StringUtils.join(licenses.iterator(), ","));
+				}
+			}
+			
+			// Set Bundle-SCM
+			if (builder.getProperty(Constants.BUNDLE_SCM) == null) {
+				List<String> scmAttributes = new ArrayList<>();
```

Comment:
Again, it seems add to make a list of string which will be joined into a string. Just start with a StringBuilder and add to it.

Reply:
done in https://github.com/bndtools/bnd/pull/2697/commits/215a4367f7a673488bdc97bb37629640cfb804e4